### ISSUE TITLE
cpplint error when I need a header to be included before any other header

### ISSUE THREAD

**Issue by shankartnj** [cpplint/cpplint#160]
I have a header which in certain files needs to be ahead of some other headers. But cpplint complains as follows: 

>  Found C system header after other header. Should be: V
iew.h, c system, c++ system, other.  [build/include_order] [4]

and in certain cases:

> Found C++ system header after other header. Should be
: TorqRootViewWin10.h, c system, c++ system, other.  [build/include_order] [4]

I thought maybe // NOLINT could work, but it looks like it only works on errors in the specific line in which it is added, in this case the errors are in the following lines and not on the line of the header i included ahead of others. The other option I have is to use // NOLINT in all the other lines following my header. But that feels dirty. Is there any other way of doing this?

**Comment by tkruse** [comment]
No, the NOLINT syntax is quite limited, also see https://github.com/google/styleguide/issues/31.

The error category can be removed in the command line options to never check include order, if your project does not want to follow the order required by cpplint.

I cannot tell from your error description whether you believe cpplint is wrong or inconsistent about it's warning, or whether you just wish to order your headers in a different way than cpplint.
